.github/workflows/build-cmake-conan.yml

```diff
@@ -36,8 +36,9 @@ jobs:
           python -m pip install conan
           conan config init
           conan config set log.print_run_commands=True
-          conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
-          conan remote add madebr_ps2dev https://api.bintray.com/conan/madebr/ps2dev
+          conan config set general.revisions_enabled=1
+          conan remote add bincrafters https://bincrafters.jfrog.io/artifactory/api/conan/public-conan
+          # conan remote add madebr_ps2dev https://api.bintray.com/conan/madebr/ps2dev
```

Comment:
```suggestion
```
You can remove it, it won't come back.
I will have to submit an official `ps2toolchain-ps2dev` and `ps2toolchain-ps2sdk` recipe to the conan central index.